File: a3_linear_search/hw_tasks_sol/fizzbuzz.py
n = 100
for i in range(n):
    if i % 3 == 0 and i % 5 == 0:
        print(i, "FizzBuzz")
    elif i % 5 == 0:
        print(i, "Buzz")
    elif i % 3 == 0:
        print(i, "Fizz")
    else:
        print(i)

File: a3_linear_search/hw_tasks_sol/fizzbuzz.cpp
#include <iostream>

using namespace std;

int main() {
	int n = 100;
	for (int i = 0; i < n; ++i) {
		if (i % 3 == 0 && i % 5 == 0) {
			cout << i << " FizzBuzz\n";
		} else if (i % 5 == 0) {
			cout << i << " Buzz\n";
		} else if (i % 3 == 0) {
			cout << i << " Fizz\n";
		} else {
			cout << i << '\n';
		}
	}

	return 0;
}
